cores/epoxy/Arduino.cpp

```diff
@@ -65,6 +65,16 @@ unsigned long micros() {
   return us;
 }
 
+void tone(uint8_t _pin, unsigned int frequency, unsigned long duration)
```

Comment:
nit: change this to
```C++
void tone(uint8_t /*_pin*/, unsigned int /*frequency*/, unsigned long /*duration*/) {}
```
because
1) The compiler will warn about unused variables otherwise.
2) Curly brace on previous line, for consistency. (I know the Arduino code base has wildly inconsistent coding style and other ugliness. I try to keep my code somewhat consistency.)